### ISSUE TITLE
I can not get  the same result in run zhifu code  

### ISSUE THREAD

**Issue by jetli6** [burness/tensorflow-101#11]
hi I download your code while it can not run, because the queue runner is always for waitting . so I change the code a little, for examlpe:
`def input_pipeline(self, batch_size, num_epochs=None, aug=False):

        image_batch=[]
        label_batch=[]
        for i in range(batch_size):
            img_path=self.image_names[self.qindex]
            img=Image.open(img_path)
            array = np.array(img)
            array.resize((28,280,1))
            image_batch.append(array)
            label_batch.append(self.labels[self.qindex])
            self.qindex = self.qindex + 1
            if(self.qindex == self.image_lenth):
                self.qindex =0
        image_batch=np.array(image_batch)
        label_batch=np.array(label_batch)
        return image_batch, label_batch`
the train step is 20000
the part of result 
[test] the step 14801 accuracy 0.7265625 spend time 10.785616874694824
the loss :  1.21366
========Begin eval stage =========
[test] the step 14901 accuracy 0.6484375 spend time 11.592663049697876
the loss :  1.03953
========Begin eval stage =========
[test] the step 15001 accuracy 0.6171875 spend time 12.94074010848999
the loss :  1.27879
========Begin eval stage =========
[test] the step 15101 accuracy 0.671875 spend time 11.467655897140503
the loss :  0.966799
========Begin eval stage =========
[test] the step 15201 accuracy 0.6640625 spend time 12.628722190856934
the loss :  1.12343
========Begin eval stage =========
[test] the step 15301 accuracy 0.6796875 spend time 12.736728429794312
the loss :  1.43497
========Begin eval stage =========
[test] the step 15401 accuracy 0.6875 spend time 12.109692573547363
the loss :  1.17038
========Begin eval stage =========
[test] the step 15501 accuracy 0.59375 spend time 12.125693559646606
the loss :  1.19976
========Begin eval stage =========
[test] the step 15601 accuracy 0.6796875 spend time 14.316818952560425
the loss :  1.20551
========Begin eval stage =========
[test] the step 15701 accuracy 0.625 spend time 13.745786190032959
the loss :  0.996233
========Begin eval stage =========
[test] the step 15801 accuracy 0.609375 spend time 22.57629132270813
the loss :  1.08622
========Begin eval stage =========
[test] the step 15901 accuracy 0.6328125 spend time 17.0779767036438
the loss :  0.936256
========Begin eval stage =========
[test] the step 16001 accuracy 0.65625 spend time 15.478885412216187
the loss :  1.23139
========Begin eval stage =========
[test] the step 16101 accuracy 0.59375 spend time 21.262216091156006
the loss :  0.893639
========Begin eval stage =========
[test] the step 16201 accuracy 0.625 spend time 19.125093936920166
the loss :  1.02303
========Begin eval stage =========
[test] the step 16301 accuracy 0.6796875 spend time 18.563061714172363
the loss :  1.05793
========Begin eval stage =========
[test] the step 16401 accuracy 0.6484375 spend time 12.70172643661499
the loss :  1.01296
========Begin eval stage =========
[test] the step 16501 accuracy 0.671875 spend time 11.184639692306519
the loss :  1.08142
========Begin eval stage =========
[test] the step 16601 accuracy 0.671875 spend time 11.707669734954834
the loss :  1.10822
========Begin eval stage =========
[test] the step 16701 accuracy 0.6875 spend time 12.335705518722534
the loss :  0.899556
========Begin eval stage =========
[test] the step 16801 accuracy 0.671875 spend time 12.702726602554321
the loss :  0.931046
========Begin eval stage =========
[test] the step 16901 accuracy 0.6796875 spend time 12.334705352783203
the loss :  1.01363
========Begin eval stage =========
[test] the step 17001 accuracy 0.640625 spend time 11.968684673309326
the loss :  0.741938

Though the code can run ,but the result  leave much to be desired。the accucess of train process  the hightest is 72% and the loss about 1.21366。  Hope you can point out  what's the problem here or  how to fix the problem of queue runner . Thank you very much!
